### PR TITLE
chore: node 20 compatibility

### PR DESCRIPTION
**Pull request by jerotire**
## Description

Updated dependencies using Node 20.

Installed, updated and tested using:

    node v20.12.0
    npm 10.5.0

CI workflow uses latest tagged release (v3.5.0) of dvsa nodejs workflow actions.

Related issue: https://dvsa.atlassian.net/browse/VOL-5346

## Before submitting (or marking as "ready for review")

- [x] Does the pull request title follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification?
- [x] Have you performed a self-review of the code
- [x] Have you have added tests that prove the fix or feature is effective and working
- [ ] Did you make sure to update any documentation relating to this change?

### CODE REVIEW

**Review by fibble — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -7,13 +7,17 @@ on:
 
 jobs:
   lint:
-    uses: dvsa/.github/.github/workflows/nodejs-lint.yaml@v3.1.0
+    uses: dvsa/.github/.github/workflows/nodejs-lint.yaml@v3.5.0
```
Comment (JoshuaLicense) [inline .github/workflows/ci.yml:10]: Could we bump to `v4.0.1` instead to bring in the Node 20 version?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -7,13 +7,17 @@ on:
...
+    with:
+      node-version: 20.x
```
Comment (JoshuaLicense) [inline .github/workflows/ci.yml:12]: Minor and feel free to resolve but we can omit this part if we update the `.nvmrc` as `setup-node` and will use that instead.

**Review by JoshuaLicense — CHANGES_REQUESTED** [review]
Needs a version bump in a few more files: `template.yaml`, `.nvmrc`, `serverless/serverless.yml` & `webpack.common.js`.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (JoshuaLicense) [reply .github/workflows/ci.yml:12]: Ah, that's not ideal 😞  Let's keep it explicit then.

**Review by JoshuaLicense — APPROVED** [review]

**Pull request merged**